client/src/components/Button/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classes from './style.css'
+import {
+  faArrowLeft,
+  faArrowRight,
+  faCheckCircle
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
```

Comment:
from line 3 to line 8 should be at the top of import the style with a line breaker

Reply:
Done